### PR TITLE
Feature/complex tab header

### PR DESCRIPTION
**Pull request by alisonailea**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [ ] Edge (Win 10)
- [x] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [x] One core team UX approval

http://docspot.devnxs.net/projects/lucid/feature-ComplexTabHeader/#/components/Tabs

### CODE REVIEW

File: src/styles/mixins.less [hunk 1]
```diff
@@ -1,3 +1,16 @@
+.clearfix() {
```
Comment (jondlm) [inline src/styles/mixins.less]: Is this being used anywhere?

File: src/components/Tabs/examples/stateful-with-complex-title-as-child.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```
Comment (jondlm) [inline src/components/Tabs/examples/stateful-with-complex-title-as-child.jsx]: Formatting looks funky on this file.

**Comment by jondlm** [comment]
Not sure if it's just me, but the tabs don't look right anymore

image

File: src/styles/mixins.less (hunk 1 above)
Comment (alisonailea) [inline src/styles/mixins.less]: removing

File: src/components/Tabs/examples/stateful-with-complex-title-as-child.jsx (hunk 2 above)
Comment (alisonailea) [inline src/components/Tabs/examples/stateful-with-complex-title-as-child.jsx]: agreed. fixed

**Comment by alisonailea** [comment]
http://docspot.devnxs.net/projects/lucid/feature-ComplexTabHeader/#/components/Tabs

**Comment by alisonailea** [comment]
screen shot 2016-07-27 at 10 57 53 am

**Comment by jondlm** [comment]
@alisonailea Awesome, that looks much better!

**Comment by ogupte** [comment]
lately we've been numbering our examples files so they always show up in the correct order: eg.: `01.stateful-with-complex-title-as-child.jsx`

File: src/components/Tabs/Tabs.jsx [hunk 3]
```diff
@@ -92,6 +92,12 @@ const Tabs = createClass({
 		 * steps.
 		 */
 		isProgressive: bool,
+
+		/**
+		 * Set the multiline className. This is typically used for styling the Tab.Title bar
+		 * for improved readability when there are multiple React elements in the tab headers.
+		 */
+		hasMultilineTitle: bool,
```
Comment (sodiumjoe) [inline src/components/Tabs/Tabs.jsx]: would it ever make sense for this to be per tab?

**Comment by sodiumjoe** [comment]
this looks great!

**Pull request merged**